t/graphics/xcolors.tex

```diff
@@ -246,12 +246,19 @@ \section{Color Compararisons}
 \end{testcolors}
 \endgroup
 
-\section{Boxes}
+\section{Setting and Using Boxes}
 \newbox\mybox
-\setbox\mybox=\hbox{Should NOT be Red!}
+{\color{red}\global\setbox\mybox=\hbox{Black on White!}}
+\copy\mybox
+
+\setbox\mybox=\hbox{{\color{black}\colorbox{white}{Black on White!}}}
 \copy\mybox
-\setbox\mybox=\hbox{Should be Red!}
-{\color{red}\copy\mybox}
+
+\setbox\mybox=\hbox{Red on Green!}
+{\color{red}\colorbox{green}{\copy\mybox}}
+
+\setbox\mybox=\hbox{Red on Green, but \color{green}\colorbox{blue}{Green on Blue!}}
+{\color{red}\colorbox{green}{\copy\mybox}}
```

Comment:
I suggest adding a test where the color is set outside of the box, before `\setbox`, and likewise before using `\copy`.

Reply:
done!